lib/utils/export/to-zip.js

```diff
@@ -0,0 +1,21 @@
+import JSZip from 'jszip';
+
+export const noteExportToZip = notes => {
+	const zip = new JSZip();
+
+	notes
+		.activeNotes
+		.forEach(
+			( { id, content } ) => zip.file( `${ id }.txt`, content )
```

Comment:
changed naming in e29452f using the following algorithm:

 - strip out all unsafe characters from each line of the note
 - trim each line of the note
 - filter out lines that are now blank
 - take the first line, or `untitled` if the note is blank after these transformations
 - limit line to 40 characters
 - save as `.txt` unless there are duplicates, in which case we append `-(1)` or `-(2)` etc…